examples/pipelines/controlnet-interior-design/components/generate_prompts/src/main.py

```diff
@@ -114,11 +114,23 @@ def load(self) -> dd.DataFrame:
 
         df = dd.from_pandas(pandas_df, npartitions=1)
 
-        index_list = [idx for idx in range(len(df))]
-        source_list = ["seed" for _ in range(len(df))]
+        # TODO remove, just use a tiny df for testing purposes
+        data = {
+            "prompts_text": [
+                "comfortable bathroom, art deco interior design",
+                "comfortable bathroom, bauhaus interior design",
+            ]
+        }
+        pandas_df = pd.DataFrame.from_dict(data)
+        df = dd.from_pandas(pandas_df, npartitions=1)
+        # end of TODO
+
+        # add id and source columns
+        df["id"] = df.assign(id=1).id.cumsum()
```

Comment:
We should use the LAION ids here.

Reply:
Can I leave this for a follow-up PR? Would like to first get all components get up and running